owncloudApp/build.gradle

```diff
@@ -175,6 +185,15 @@ android {
 
     buildFeatures {
         viewBinding true
+        compose true
+    }
+
+    composeOptions {
+        kotlinCompilerExtensionVersion = "1.5.5"
+    }
+
+    kotlinOptions {
+        jvmTarget = "1.8"
```

Comment:
With the previous `=` comment, I meant all of these as well. The syntax is the same in the whole file

Reply:
In `jvmTarget` , the `=`  is necessary because the app doesn't compile without it. On the other hand, in `kotlinCompilerExtensionVersion` is unnecessary.